aws-kendra-index/src/main/java/software/amazon/kendra/index/CreateHandler.java

```diff
@@ -41,17 +45,17 @@ public class CreateHandler extends BaseHandlerStd {
             // Set the delay to two minutes so the stabilization code only calls
             // DescribeIndex every two minutes - create takes
             // 30/45+ minutes so there's no need to check the index is active more than every couple minutes.
-            .delay(Duration.ofMinutes(2))
+            .delay(Duration.ofMinutes(10))
```

Comment:
Can you update the comments?

Reply:
Also, 10 mins seem to rather long, any rational on why increase this to 10 minutes? This could means that the index create waittime can increase by 10 mins in the worst case.

Reply:
Ah, looks like I pulled this as part of internal changes. I'll revert for now. We can re-introduce is necessary.